bin/upload-dist-staging.sh

```diff
@@ -45,6 +51,38 @@ function package_has_changes() {
     fi
 }
 
+# kubernetes packages before 1.24 are not available in the new repo, and so should be copied
+function is_old_kubernetes() {
+    local package="$1"
+    if echo "${package}" | grep -q "kubernetes-1.15" ; then
+        return 0
+    fi
+    if echo "${package}" | grep -q "kubernetes-1.16" ; then
+        return 0
+    fi
+    if echo "${package}" | grep -q "kubernetes-1.17" ; then
+        return 0
+    fi
+    if echo "${package}" | grep -q "kubernetes-1.18" ; then
+        return 0
+    fi
+    if echo "${package}" | grep -q "kubernetes-1.19" ; then
+        return 0
+    fi
+    if echo "${package}" | grep -q "kubernetes-1.20" ; then
+        return 0
+    fi
+    if echo "${package}" | grep -q "kubernetes-1.21" ; then
+        return 0
+    fi
+    if echo "${package}" | grep -q "kubernetes-1.22" ; then
+        return 0
+    fi
+    if echo "${package}" | grep -q "kubernetes-1.23" ; then
+        return 0
+    fi
```

Comment:
wanna explicitly return 1 here?

```suggestion
    fi
    return 1
```

Reply:
yeah I didn't think that one through